staubli/package.xml

```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<package>
+<package ="2">
```

Comment:
```suggestion
<package format="2">
```

Reply:
Was working on a fix while you posted this suggestion (see next commit).  Thanks!